src/conn/conn_open.c

```diff
@@ -80,28 +80,12 @@ __wt_connection_close(WT_CONNECTION_IMPL *conn)
 	F_SET(conn, WT_CONN_CLOSING);
```

Comment:
Unrelated to your change specifically, but in the ballpark: `wt_connection_close` is only called from 2 functions in `conn_api.c`. It should move to that file and be a static in there instead of here.

Reply:
I don't disagree, but every function in `conn_open.c` is only called from `conn_api.c`, by that argument, we'd merge `conn_open.c` entirely into `conn_api.c`.

And, that's true for `conn_handle.c` as well. The files `conn_handle.c` and `conn_open.c` are 263 and 141 lines respectively. The file `conn_api.c` is 2838 lines, so I guess I see how we got here, somebody was trying to split things up.

You sure?

@agorrod, @michaelcahill, any ideas how to cleanly split up the `WT_CONNECTION` code into multiple files?

Reply:
In my ideal world conn_api.c would just contain the API entry points. Simple entry points could be implemented there directly , but otherwise the implementations should be in a different file.

i.e: I'd rather not move the functions into conn_api.c.

Reply:
Thanks for the bigger picture. Stand down, we'll leave it as-is.